src/components/FormInput/FormInput.js

```diff
@@ -6,22 +11,19 @@ import Label from './Label';
 import ErrorMessage from './ErrorMessage';
 import Input from './Input';
 
-const FormInput = ({ name, label, placeholder, type, ...props }) => {
+const FormInput = ({ name, label, children, icon: Icon, type = 'text', component: Component = Input, ...props }) => {
```

Comment:
Why do you add types for `icon` and `component`?

Reply:
It is not type :) It's alias for parameter so later I can write `<Icon {...something} />` since for react in JSX it actually matters if it is lower case or upper case name for element.